NEWS

```diff
@@ -5,6 +5,7 @@ PHP                                                                        NEWS
 - Core:
   . Fixed bug GH-12953 (false positive SSA integrity verification failed when
     loading composer classmaps with more than 11k elements). (nielsdos)
+  . Fixed bug GH-12999 zend_strnlen build when strnlen is unsupported. (rainerjung)
```

Comment:
Silly remark: you forgot the () around the text. May or may not cause issues with the NEWS conversion script the website uses :shrug:

Reply:
oh I did not know that thanks.